api/src/domain/annual-filings/calculateNextAnnualFilingDates.test.ts

```diff
@@ -13,46 +13,82 @@ jest.mock("@shared/dateHelpers", () => {
 });
 const currentDateMock = (getCurrentDateModule as jest.Mocked<typeof getCurrentDateModule>).getCurrentDate;
 
-describe("calculateNextAnnualFilingDate", () => {
+describe("calculateNextAnnualFilingDates", () => {
   describe("when today is October 5, 2021", () => {
     beforeEach(() => {
       currentDateMock.mockReturnValue(dayjs("2021-10-05", defaultDateFormat));
     });
 
     it("finds the next-year date for an earlier-this year date", () => {
-      expect(calculateNextAnnualFilingDate("2021-03-01")).toEqual("2022-03-31");
+      expect(calculateNextAnnualFilingDates("2021-03-01")).toEqual([
```

Comment:
I think this (and all of the test names in here) should probably be changed from "date" to "dates" or "next 3 years dates" or something like that

Reply:
Makes sense. Included!